packages/main/src/MenuItem.ts

```diff
@@ -169,6 +183,12 @@ class MenuItem extends ListItem implements IMenuItem {
 	@property({ type: Boolean, noAttribute: true })
 	_siblingsWithIcon = false;
 
+	/**
+	 * Keeps the previous selected state of the item.
+	 */
+	@property({ noAttribute: true })
+	_prevSelected?: boolean;
```

Comment:
why is this needed?

Reply:
It seems to be a redundand/forgotten code piece that is not used anymore. Removed.